libs/core/src/components/sage-avatar/sage-avatar.scss

```diff
@@ -0,0 +1,79 @@
+:host {
+  /**
+    * @prop --size: Custom size of avatar. Setting this will *not* override a set `size` prop value.
+  */
+
+  --size: var(--sage-spacing-2xl);
+
+  display: inline-block;
+}
+
+$sage-avatar-sizes: (
+  xs: var(--sage-spacing-md),
+  sm: 32px,
+  md: 40px,
+  lg: 56px,
+  xl: var(--sage-spacing-2xl)
+);
+
+@each $size, $value in $sage-avatar-sizes {
+  :host(.sage-avatar--#{$size}) div {
+    height: $value;
+    width: $value;
+  }
+}
+
+:host(.sage-avatar--admin) div {
+  border-radius: 12px;
```

Comment:
May want to check with design here. I was seeing some usage of 12 in other components and it turned out it was incorrect. Would be great if they'd agree to one of the token sizes. Definitely not a show-stopper.